i18n/id/white-paper.md

```diff
@@ -0,0 +1,581 @@
+* $\gamma(\psi)$ sesuai dengan insentif bagi orang-orang untuk mempertaruhkan jaringan.
+Saat $\psi$ meningkat, lebih sedikit hadiah $\gamma(\psi)$ diperlukan untuk mengamankan jaringan.
+
+Inflasi tahunan $I$ akan setara dengan $(\tau + \gamma + \delta)$ dan dihitung sebagai berikut:
+
+$$
+I = \frac{\textrm{persediaan token di akhir tahun} - \textrm{persediaan token di awal tahun}}{\textrm{persediaan token di awal tahun}} = (\tau + \gamma + \delta)
+$$
+
+Kontribusi terhadap inflasi $\tau_{\textsc{all}}$ (insentif yang didistribusikan ke semua paket yang lebih curam) dan $\gamma_{\textsc{all}}$ (insentif yang didistribusikan ke semua kontributor untuk keamanan jaringan) harus dipertimbangkan untuk memastikan bahwa sistem mendorong rasio seduhan/staking yang optimal.
+
+Saat kami fokus pada insentif yang didistribusikan di semua paket yang lebih curam, kami menentukan bahwa
+$\tau_{\textsc{all}}$
+adalah fungsi dari rasio seduhan $\chi$ dan oleh karena itu
+$\tau_{\textsc{all}}(\chi) = \chi \cdot \tau(\chi)$.
+Dari analisis kami sebelumnya, kami dapat melihat bahwa
+$\tau_{\textsc{semua}}(\chi_{ideal}) = \chi_{ideal} \cdot \tau_{ideal}$.
+Karena tujuannya adalah untuk mencapai keadaan di mana
+$\chi = \chi_{ideal}$
+, hadiah
+$\tau_{ideal}(\chi)$
+harus maksimal pada nilai tersebut.
+
+Misalkan $\tau_{ideal} = \tau(\chi_{ideal})$
+menjadi tingkat hadiah yang diberikan oleh jaringan pada skenario ideal di mana
+$\chi = \chi_{ideal}$.
+
+Biarkan $\tau_{0}$ menjadi batas $\tau_{\textsc{all}}(\chi)$ karena $\chi$ menjadi nol ketika tidak ada anggota komunitas teh yang mengambil paket apa pun.
+Nilai $\tau_{0}$ harus mendekati nol tetapi tidak nol untuk mendorong pengadopsi awal.
+Seperti yang disarankan oleh penelitian Yayasan web3, kami mengusulkan bahwa:
+
+* fungsi inflasi tumbuh secara linier antara $\chi = 0$ dan $\chi = \chi_{ideal}$, dan
+* itu meluruh secara eksponensial antara $\chi = \chi_{ideal}$ dan $\chi = 1$.
+
+Kami memilih penurunan eksponensial serupa untuk $\tau_{\textsc{all}}(\chi)$ karena ini menyiratkan penurunan eksponensial $\tau(\chi)$, dan kami ingin imbalan turun tajam melampaui $\chi_{ ideal}$ untuk mencegah satu paket menerima semua hadiah.
+
+Peluruhan didefinisikan sehingga tingkat inflasi berkurang paling banyak 50% ketika $\chi$ menggeser unit $d$ ke kanan $\chi_{ideal}$ – yaitu.
+$\tau_{\textsc{semua}}(\chi_{ideal} + d) \geq \tau_{\textsc{semua}} \cdot 0.5$.
+
+Kami mengusulkan fungsi tingkat bunga dan tingkat inflasi berikut, yang bergantung pada parameter $\chi_{ideal}$, $\tau_{ideal}$, $\tau_{0}$ dan $d$.
+
+\bemula{selaraskan*}
+&\tau_{\textsc{semua}}(\chi) = \tau_{0} + (\tau_{\textsc{semua}}(\chi_{ideal}) - \tau_{0})\frac{\chi }{\chi_{ideal}}\enspace\textrm{for}\;0 < \chi \leq \chi_{ideal} \\
+&\tau_{\textsc{semua}}(\chi) = \tau_{0} + (\tau_{\textsc{semua}}(\chi_{ideal}) - \tau_{0}) \cdot 2^{ (\chi_{ideal}-\chi)/d}\enspace\textrm{for}\;\chi_{ideal} < \chi \leq 1
+\end{selaras*}
```

Comment:
Please check the formulas again. You have translated some of the subscript like _all_ and _ideal_ inside the formula that shouldn't be translated. 

I didn't check everywhere in the white-paper. But please make sure you don't translate formulas. 

Please refer to the locations in the original file here: https://github.com/teaxyz/white-paper/blob/main/white-paper.md?plain=1#L471 to make sure you are not overwriting any formula.